app/database.py

```diff
@@ -0,0 +1,11 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+SQLALCHEMY_DATABASE_URL = "postgresql://postgres:admin@localhost/test_db"
```

Comment:
Connection strings should never be exposed like this. We can use environment variables for this.
This is not just about username and password, but all the secrets that should be secured from external users.

Reply:
> Connection strings should never be exposed like this. We can use environment variables for this. This is not just about username and password, but all the secrets that should be secured from external users.

Will get on fixing that right away.